src/Microsoft.TemplateEngine.Cli/LocalizableStrings.resx

```diff
@@ -347,7 +347,7 @@ See https://aka.ms/dotnet-install-templates to learn how to install additional t
     <value>The following templates partially match the input. Be more specific with the template name and/or language.</value>
   </data>
   <data name="TemplateNotValidGivenTheSpecifiedFilter" xml:space="preserve">
-    <value>{0} matches the specified name, but has been excluded by the --type parameter. Remove or change the --type parameter to use that template.</value>
+    <value>{0} template partially matched, but has been excluded by the --type parameter. Remove or change the --type parameter to use that template.</value>
```

Comment:
Do we need the arg `{0}` since we already know that there is just "one" template here?

Or do we really need two strings here or can we just write `{0} template(s) partially matched,...` ? How is this situation handled in the rest of the localized strings?

Reply:
@bozturkMSFT  I already asked this question before: we don't have any strings with "have/has" or any verbs with changing forms.  For plurals several strings just have (s). I think it makes sense to ask Kathleen probably to rephrase it to avoid have/has or use have(has).

Reply:
I suppose that one string is better than two. This string with have/has is the old one, just leave it be and add your own localized string. Kathleen already provided us the one without this problem: `12 template(s) partially matched, but failed on <whatever>`, where `<whatever>` should be `type=item`, for example.  `<whatever>` should be a parameter for the localized string, like in "No templates found matching" string. Later we will be able add support for the lang option there.

Reply:
Changed to `{count} template(s) partially matched, but failed on {reason}`